src/Operations.jl

```diff
@@ -116,11 +116,18 @@ end
 function update_manifest!(ctx::Context, pkgs::Vector{PackageSpec}, deps_map)
     manifest = ctx.env.manifest
     empty!(manifest)
+    if ctx.env.pkg !== nothing
+        pkgs = [pkgs; ctx.env.pkg]
```

Comment:
Should this filter out if it already is in the manifest?

Reply:
Hm, probably but not sure

Reply:
Seems to disappear in some pruning, and they should (?) be equivalent since if it exists in the manifest Pkg must have put it there and then it should match.